Cloud-natural-language-api-challenge-lab-guide.html

```
@@ -0,0 +1,1328 @@
+                        <li>
+                            <code>encodingType</code
+                            >：レスポンス内のオフセット（トークンの位置情報）を計算する際の文字エンコーディング。日本語や絵文字を含む場合、<code>UTF8</code>・<code>UTF16</code>・<code>UTF32</code>のどれを選ぶかで文字位置の数え方が変わるため、扱う言語圏に応じて明示的に指定することが推奨される。
+                        </li>
+                    </ul>
+
+                    <h3>curlコマンドでの呼び出し方（ベストプラクティス込み）</h3>
+                    <pre><code class="language-bash">export API_KEY=&lt;Task1で取得したキー&gt;
+
+curl "https://language.googleapis.com/v1/documents:analyzeSyntax?key=${API_KEY}" \
+  -s -X POST -H "Content-Type: application/json; charset=utf-8" \
+  --data-binary @analyze-request.json > analyze-response.txt</code></pre>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)